src/html.htl

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html data-sly-attribute="${content.document.documentElement.attributesMap}">
 <head>
-  <title>${content.title}</title>
+  <title>${content.meta.title}</title>
```

Comment:
Is there always a `meta.title` ?

Reply:
Yes, see https://github.com/adobe/helix-pages/blob/breaking-202103/src/extract-metadata.js#L171-L173